File: Frontend/src/app/system/_domains/StockHistory.ts
import { Injectable } from '@angular/core';
import { _EntityBase } from './_EntityBase';
import { StockType } from './enums/StockType';
import { StockHitType } from './enums/StockHitType';

@Injectable()
export class StockHistory extends _EntityBase {

     public ProductName: string;
     public StockType: StockType;
     public StockHitType: StockHitType;
     public Quantity: number;
     public SupplierName: string;
     public CustomerName: string;
     public CreateDate: Date;
     public IdProduct: number;
}

File: Backend/Stock.Business/Entities/StockHistory.cs
using Stock.Domain;
using Stock.Domain.Enum;
using System;
using System.Collections.Generic;
using System.Text;

namespace Stock.Business.Entities
{
    public class StockHistory
    {
        public string ProductName { get; set; }
        public StockType StockType { get; set; }
        public StockHitType? StockHitType { get; set; }
        public decimal Quantity { get; set; }
        public string SupplierName { get; set; }
        public string CustomerName { get; set; }
        public DateTime CreateDate { get; set; }
        public int IdProduct { get; set; }
    }
}
